book/content/risk_assessment/risk_assessment.md

```diff
@@ -1,31 +1,43 @@
+| -------------|----------|------|
+|[Knowledge of Software Testing](https://www.test-institute.org/Introduction_To_Software_Testing.php) | Helpful | How it can be helpful in reducing the risk of failure |
+| Recommended skill level | beginner-intermediate | Depends on the tools |
+
+## Summary
 Use a risk assessment to help choose the appropriate sustainable software concepts for your project. Too little and your software is unsustainable; too much and you won’t be able to Get On With It. It can take just a few seconds, but gets you off on the right foot.
+Risk assessment is one of the broader areas of Risk Management which constitutes of :
+* Risk Identifiction
+* Risk Analysis
+* Risk Prioritization
 
-## Longer read…
-We all use risk assessments all the time. Sometimes they’re formal procedures to ensure an activity is safe, but most of the time they’re the thought of a moment- Is this coffee too hot? Is there a bus coming? Software is no different, and using a risk assessment approach like the one described below can really help make your work successful and sustainable.
+as its sub-parts. All the above three classifications are equally important as each one of them depends on each other. 
+1. __Risk Identification__ is a technique to detect the path where the possibilities of risks and faults are likely to occur.
+Once the risk has been identified, then only you can proceed further with the rest of the processes as without first identifying the risks you cannot go with the analysis of it.
 
-### The risk matrix
-A risk matrix is a very popular way of quantifying what’s going on with the thing you’re interested in. One axis measures exposure in some way, and the other the impact of a mishap. The further from the origin, the more safeguards are needed to make the risk acceptable.
+2. __Risk Analysis__: After the risk has been identified, it is very important to analyze before it will create any types of major issues on your project. Risk Analysis is a process of determining the consequences and meanwhile estimating the level or the extent of the risk.
 
-![Impact vs complexity risk matrix](../figures/risk_matrix.png)
+3. __Risk Prioritization__ is a methodology to interpret the impacts of the encountered risks and prioritize it accordingly on the basis of the level of damage the risk could make.
+
+The main goal of Risk Assessment is to reduce the uncertainity of the risks and to eradicate it completely. 
+The significant approach for doing this is to find the root cause which is reponsible for troubling down to risk, which are generally the bugs, errors and loop holes in the software. 
+The next step which is to be followed is to calculate the percentage error followed by prioritization.
+Risk Assesssment builds a platform to deal with all these and enables you to work on the project efficiently.
+
+
+***Risk Assessment of the software is thus vital for your project to keep it safe, protective and risk free!!*** 
```

Comment:
```suggestion
```

Reply:
It is being repeated.

Reply:
@malvikasharan as mentioned commit suggestions is not activated for me... The reason I guess which I have explained in my new comment below.
Sorry, for inconvenience.